uiowa/uiowa
# [Facilities] Artwork - added `field_gallery_images` to handle more images

This pr adds an additional gallery field if the artwork needs to display additional images.  The images are set to appear in a grid in the sidebar. 

# How to test

```
ddev blt frontend && ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local uli /node/3641/edit
```

- Code review
- Add additional images to the "Image(s)" field
- Confirm that they display under the image in square thumbnails
- Go to https://facilities.uiowa.ddev.site/featured-content-test and confirm teaser image only shows first image
- Confirm the above with only one image
- Test Artwork open graph and twitter card images are using the `field_gallery_images` field

### This pr updates how the uiowa-image-gallery class is added
- Test that article photoswipe galleries are still working on articles and image blocks and classroom room pages

```
ddev blt ds --site=classrooms.uiowa.edu && ddev drush @classrooms.local uli /abw-110
```
```
ddev blt ds --site=now.uiowa.edu && ddev drush @now.local uli /news/2023/11/look-back-iowa-womens-wrestling-blazes-new-trail
```
```
ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /2023/11/look-back-iowa-womens-wrestling-blazes-new-trail
```

## Images

### With gallery, video
<img width="1677" alt="Screenshot 2023-11-21 at 8 53 18 AM" src="https://github.com/uiowa/uiowa/assets/1036433/f4e6b35f-d9e8-40c6-b827-4a217e1ff31e">


## With no gallery, no video
<img width="1766" alt="Screenshot 2023-11-21 at 8 53 00 AM" src="https://github.com/uiowa/uiowa/assets/1036433/7b2cd370-c1e5-478b-9f81-5434ba4f8055">
